pkg/syncer/fullsync.go

```diff
@@ -720,26 +720,33 @@ func volumeInfoCRFullSync(ctx context.Context, metadataSyncer *metadataSyncInfor
 			} else if metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorWorkload &&
 				IsPodVMOnStretchSupervisorFSSEnabled {
 				pv := volumeIdTok8sPVMap[volumeID]
-				pvc, err := metadataSyncer.pvcLister.PersistentVolumeClaims(
-					pv.Spec.ClaimRef.Namespace).Get(pv.Spec.ClaimRef.Name)
-				if err != nil {
-					log.Warnf("Failed to get pvc for namespace %s and name %s. err=%+v",
-						pv.Spec.ClaimRef.Namespace, pv.Spec.ClaimRef.Name, err)
+				// claimref will be nil when volume is static provisioned and not claimed by pvc.
```

Comment:
the PV could also be released from earlier PVC associated, can mention generically that this would happen for available/released PV